model/RdsResultStorage.php

```diff
@@ -275,7 +276,7 @@ public function getVariables($callId)
             ->from(self::VARIABLES_TABLENAME)
             ->andWhere(self::CALL_ID_ITEM_COLUMN .' IN(:ids)')
             ->orWhere(self::CALL_ID_TEST_COLUMN .' IN(:ids)')
-            ->orderBy(self::VARIABLES_TABLE_ID)
+            ->orderBy(self::CREATED_AT)
```

Comment:
will this work as expected on GC Spanner?
does the newSQL handle properly the order by ?

Reply:
Yes, this is DBAL generating a SQL statement with plain "ORDER BY created_at" clause, and Spanner executes native SQL queries.